SYCL/Plugin/interop-level-zero-interop-task-mem.cpp

```diff
@@ -0,0 +1,53 @@
+// REQUIRES: level_zero, level_zero_dev_kit
+// RUN: %clangxx -fsycl -fsycl-targets=%sycl_triple %level_zero_options %s -o %t.out
+// RUN: env SYCL_BE=PI_LEVEL_ZERO %GPU_RUN_PLACEHOLDER %t.out
```

Comment:
```suggestion
// RUN: %GPU_RUN_PLACEHOLDER %t.out
```
I don't think setting `SYCL_BE` is needed here. `// REQUIRES: level_zero` should do the trick. @vladimirlaz, am I right?
Anyway use of this environment variable is deprecated.

Reply:
you are right